plugins_overview.py

```diff
@@ -112,8 +112,7 @@ def add_all_items(method, dest, wrapopen=r'\(', wrapclose=r'\)'):
 if (len(sys.argv) > 1) and (sys.argv[1] == "csv"):
     print("plugin;url;distros;profiles;packages;copyspecs;forbidden;commands;"
           "service_status;journals;env_vars")
-    for plugname in plugs_data.keys():
-        plugin = plugs_data[plugname]
+    for _, plugin in plugs_data.items():
```

Comment:
`plugname` is referred on line 125/126:

    out = ";" if line > 0 else f"{plugname};{plugin['sourcecode']}"

so we need

    for plugname, plugin in ..

Reply:
Good catch! Fixed now.